ci/authn-azure/run-authn-azure.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -ex
+
+# Get an Azure access token
+azure_access_token=$(curl \
```

Comment:
The scripts have already been merged no?

Reply:
Yes but as part of the deletion yesterday they were removed. This PR will re-add them.